simpletest/simpletest

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+#: exec_target = cli
+
+## Run automated tests with SimpleTest.
+##
+## Usage: fin simpletest <args>
+
+# Determine the correct path for the run-tests.sh script. This will depend on
+# the version of Drupal.
+if [ -f "core/scripts/run-tests.sh" ]; then
+    SCRIPT_FILE="$PROJECT_ROOT/$DOCROOT/core/scripts/run-tests.sh"
+else
+    SCRIPT_FILE="$PROJECT_ROOT/$DOCROOT/scripts/run-tests.sh"
+fi
+
+# Execute the script.
+php $SCRIPT_FILE \
+  --php `which php` \
```

Comment:
What this line is for?

Reply:
It's to set the path to the `php` executable. Though it seems to be running OK at the moment without it on D7.

Reply:
What is the need to set the path? you execute `php` in this line. It means the path is already set.